underpants.go

```diff
@@ -257,6 +277,31 @@ func serveHttpProxy(d *disp, w http.ResponseWriter, r *http.Request) {
     return
   }
 
+  if d.config.HasGroups() {
+    authorized := false
+
+AuthCheck:
```

Comment:
My, very limited, understanding of labels in Go is that they need to come before the statement you're labelling.

* http://golang.org/ref/spec#Break_statements
* http://golang.org/ref/spec#Label